fast_llm/data/dataset/gpt/sampled.py

```diff
@@ -128,6 +128,7 @@ def _sample(self) -> None:
         # Calculate basic stats.
         documents_per_epoch = document_sizes.numel()
         tokens_per_epoch = document_sizes.sum().item()
+        # TODO MTP: Produce more labels to provide labels for the multi-token prediction heads?
```

Comment:
I'm not following, could you explain what you mean here?
Surely you're shifting the sequence for the additional heads, are you not?

Reply:
Ok I can have a quick look at this this afternoon, and let's try to merge this PR afterwards

Reply:
Will merge now, and add this to a new PR, since I'd need to test it a little bit more